src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -114,6 +126,18 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
         return Promise.resolve();
     }
 
+    private renderMessages(amount: number, containerId: string) {
+        const { paypal } = this._window;
+        if (!paypal || !paypal.Messages) {
+            return;
+        }
+
+        return paypal?.Messages({
```

Comment:
And here you don't need to check if paypal is not empty, because you did it in previous if statement

Reply:
fixed